package-lock.json

```diff
@@ -73,7 +72,6 @@
 			"dependencies": {
 				"anymatch": "~3.1.1",
 				"braces": "~3.0.2",
-				"fsevents": "~2.1.2",
```

Comment:
That's weird

Reply:
no idea -- anything to do?

Reply:
Yeah chokidar definitely depends on fsevents, so let's just revert this lockfile and then I'll investigate later